raw/yant.csv

```diff
@@ -0,0 +1,119 @@
+#02,white,255,255,255,KintaroOe
```

Comment:
Do you mind dropping that `#` or maybe using `Y` for "yant" maybe?

```suggestion
02,white,255,255,255,KintaroOe
```

```suggestion
Y02,white,255,255,255,KintaroOe
```

Reply:
> Do you mind dropping that `#` or maybe using `Y` for "yant" maybe?

I simply used their numbering mode. I think there is no problem dropping the # or substitute it with the Y.